examples/01_basic_agent.py

```diff
@@ -57,6 +62,21 @@ async def get_capital_info(city_input: CityInput) -> Run[CapitalOutput]:
     ...
 
 
+async def display_completions(run: Run[T]) -> None:
```

Comment:
```suggestion
async def display_completions(run: Run[Any]) -> None:
```

no need to have a variable here